.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /Cargo.lock
+.idea/
```

Comment:
Suggest creating a global .gitignore file and putting this in it. It's a good way to prevent your IDE files from ending up in repos. See this: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.
```suggestion
```